content/gettingstarted/index.adoc

```diff
@@ -39,89 +39,66 @@ icon:lightbulb[size=1x]{nbsp}For several commands listed on this page, the OpenS
 
 == Installing Kiali
 
-=== Preparation
+Kiali can be installed in an OpenShift or Kubernetes cluster environment. For OpenShift only, a minimal preparation is necessary.
 
-Kiali can be installed in an OpenShift or Kubernetes cluster environment. Before you begin, you may need to prepare your OpenShift or Kubernetes environment.
+=== OpenShift Preparation
 
-==== OpenShift Preparation
-
-If you are installing on OpenShift, you need to grant the `cluster-admin` role to the user that is installing Istio and Kiali.  If OpenShift is installed locally on the machine you are using, the following command should log you in as user `system:admin` which has this `cluster-admin` role:
+If you are installing on OpenShift, you need to grant the `cluster-admin` role to the user that is installing Istio and Kiali. If OpenShift is installed locally on the machine you are using, the following command should log you in as user `system:admin` which has this `cluster-admin` role:
 
 [source,bash]
 ----
 oc login -u system:admin
 ----
 
-==== Kubernetes Preparation
-
-If you are not installing on OpenShift but instead installing on Kubernetes, there are no additional preparation steps required unless you are installing on Minikube. When installing on Minikube, ensure that you enable the Ingress add-on by executing the following command:
-
-[source,bash]
-----
-minikube addons enable ingress
```

Comment:
I haven't tried minikube for a while. Is this not needed anymore?

Reply:
I wasn't able to make it work with just "minikube addons enable ingress". While looking at the minikube doc for the nginx ingress, it seemed some other commands were necessary too - but did not work either for me. Personally I prefer to use kubectl port-forward.

So, I don't know, my take is that we can remove that kind of opinionated how-to because we don't check constantly if it continues to work with version x.y.z of minikube or kubernetes, and instead we just provide some links to external doc and/or assume the user already knows better than us how he or she wants to manage ingresses.